src/main/sensors/gyro.c

```diff
@@ -1120,9 +1129,39 @@ FAST_CODE void gyroUpdate(timeUs_t currentTimeUs)
         gyroUpdateSensor(&gyroSensor1, currentTimeUs);
         gyroUpdateSensor(&gyroSensor2, currentTimeUs);
         if (isGyroSensorCalibrationComplete(&gyroSensor1) && isGyroSensorCalibrationComplete(&gyroSensor2)) {
-            gyro.gyroADCf[X] = (gyroSensor1.gyroDev.gyroADCf[X] + gyroSensor2.gyroDev.gyroADCf[X]) / 2.0f;
-            gyro.gyroADCf[Y] = (gyroSensor1.gyroDev.gyroADCf[Y] + gyroSensor2.gyroDev.gyroADCf[Y]) / 2.0f;
-            gyro.gyroADCf[Z] = (gyroSensor1.gyroDev.gyroADCf[Z] + gyroSensor2.gyroDev.gyroADCf[Z]) / 2.0f;
+            switch(gryoForX) {
```

Comment:
Loop over axes

Reply:
@ledvinap I've been instructed by @mikeller to use a switch if you are comparing single variables. The performance is better using a switch.

Reply:
Switch is fine, but it should be wrapped in for loop over axes